syntax = "proto3";

package pb;

option go_package = "./pb";

service SQLInfoService {
  rpc SendSQLInfo (SQLInfoRequest) returns (SQLInfoResponse);
}

message SQLInfoRequest {
  string sql_text = 1;
  int32 txn_id = 2;
  repeated int32 keys = 3;
  repeated int32 region_ids = 4;
}

message SQLInfoResponse {
  bool success = 1;
}
